### ISSUE TITLE
Standardize detail pages

### ISSUE THREAD

**Issue by JessicaDierking** [hpi-swt2/compass-portal-blue#167]
As a user, I want to have standardized detail pages for persons, rooms, buildings, and locations with the same layout.

DoD
- [ ]  Tests
- [ ]  Documentation
- [ ]  CI Build passing
- [ ]  Peer Review
- [ ]  Approval from PO

**Comment by hendrix15** [comment]
This goes with standardized (one step-) "back" and "back to map" buttons. (There are some pages that you can't leave).

**Comment by hendrix15** [comment]
We should add a UI sketch here

**Comment by hendrix15** [comment]
@Olliwehr did you already rework the layout for all the detail pages?
@JessicaDierking as I think that Olli already worked on that, do we need any more changes in the layout? If so, we really need to add a sketch (we should do it together). If not, I can create a new issue only for the back buttons and close this one.

**Comment by Olliwehr** [comment]
@hendrix15 yeah, @philipraetsch and I worked on some show pages (those which get displayed in the search results on the main Map Page) with #131. Mainly, we standardized them to each have a header containing the name of the shown entity and tweaked a bit styling wise (images, etc.).
But I'm not sure whether these changes display the current state of the show-pages, as these we are also changed a bit afterwords with #155 by the APMW team.

Regarding the back buttons: i think that was something we discussed about, that these would probably need an own issue. As its more like a more general thing to consider, not only for the few show pages we touched in the mentioned issue 🤔
